stix2validator/output.py

```diff
@@ -140,30 +140,26 @@ def print_results(results):
     """Prints `results` (the results of validation) to stdout.
 
     Args:
-        results: A dictionary of ValidationResults instances. The key is the
-            file path to the validated document.
+        results: A list of FileValidationResults instances.
 
     """
-    if not hasattr(results, 'items'):
-        results_arr = {'': results}
-        results = results_arr
-
-    level = 0
-    for fn, result in sorted(iteritems(results)):
```

Comment:
This code would have sorted tuples.  That sorts in lexicographical order, by tuple element.  There wasn't a sort order defined (was there?) on ValidationResults (the second element of each tuple), so I didn't think this was was well defined.  So I dropped the sorting.

Reply:
I think sorting by filename is a good idea. The filename should be unique, so the ordering on ValidationResults shouldn't (?) be an issue. I'd also be ok if results was a dictionary mapping filename to results, if sorting the keys in that dictionary made more sense.